contracts/scripts/files/mainnetConfig.json

```diff
@@ -0,0 +1,46 @@
+{
+  "contracts": {
+    "l1Timelock": "0xE6841D92B0C345144506576eC13ECf5103aC7f49",
+    "rollup": "0x5eF0D09d1E6204141B4d37530808eD19f60FBa35",
+    "bridge": "0x8315177aB297bA92A06054cE80a67Ed4DBd7ed3a",
+    "sequencerInbox": "0x1c479675ad559DC151F6Ec7ed3FbF8ceE79582B6",
+    "rollupEventInbox": "0x57Bd336d579A51938619271a7Cc137a46D0501B1",
+    "outbox": "0x0B9857ae2D4A3DBe74ffE1d7DF045bb7F96E4840",
+    "inbox": "0x4Dbd4fc535Ac27206064B68FfCf827b0A60BAB3f",
+    "osp": "0x3E1f62AA8076000c3218493FE3e0Ae40bcB9A1DF"
+  },
+  "proxyAdmins": {
+    "outbox": "0x554723262467f125ac9e1cdfa9ce15cc53822dbd",
+    "bridge": "0x554723262467f125ac9e1cdfa9ce15cc53822dbd",
+    "rei": "0x554723262467f125ac9e1cdfa9ce15cc53822dbd",
+    "seqInbox": "0x554723262467f125ac9e1cdfa9ce15cc53822dbd"
+  },
+  "settings": {
+    "confirmPeriodBlocks": 50400,
```

Comment:
Arb1 is using a value of [45818](https://etherscan.io/address/0x5eF0D09d1E6204141B4d37530808eD19f60FBa35#readProxyContract#F7) (assuming 13.2 s block time)
50400 is more correct but we might want to change it separately

Reply:
I think lets change it now - it's less overhead and we can include it in the AIP